src/pages/sweepstake/mintform/index.jsx

```diff
@@ -0,0 +1,288 @@
+          .mint(tokenUri, token, duration * 60)
+          .estimateGas()
+        
+        let res = await contractERC721EsolidarSweepstake.methods
+          .mint(tokenUri, token, duration * 60)
+          .send({ from: address, gasLimit })
+
+        sweetAlertSuccess('Your campaign was created successfully!')
+
+        setTimeout(() => {
+          window.location.href = `/sweepstake/${res.events.Transfer.returnValues.tokenId}`
+        }, 2000)
+      })
+
+    } catch (e) {
+      
+      // If Generic error
+      if(e.message == 'Internal JSON-RPC error.'){
+        setErrorMessage('Error interacting with the smart contract')
+      }
+
+      if(e.message){
+        setErrorMessage(e.message)
+        return
+      }
+      if(e.data?.message){
+        setErrorMessage(e.data.message)
+        return
+      }
+      if(e.data?.reason){
+        setErrorMessage(e.reason)
+        return
+      }
+      if(e.reason){
+        setErrorMessage(e.reason)
+        return
+      }
+
+      // Generic error
+      setErrorMessage('Error interacting with the smart contract')
+    }
+  }
+
+  return (
+
+    <form className="mint-form" onSubmit={e => e.preventDefault()}>
+      <Navbar/>
+      <div className="flex flex-col md:flex-row gap-20 justify-center">
+        <div>
+          {/* ===> FILE  */}
+          {/* ===> PREVIEW FILE  */}
+          <div className="mb-5 flex w-full items-center justify-center md:justify-end">
+            {file && (
+              <div className="upload-preview">
+                {file.type == 'video/mp4' || file.type == 'video/ogg' ? (
+                  <ReactPlayer className="ReactPlayerInput" width="100%" controls height="100%" url={mediaSrc} />
+                ) : file.type == 'audio/mp3' || file.type == 'audio/mpeg' ? (
+                  <ReactAudioPlayer src={mediaSrc} controls />
+                ) : (
+                  <img src={urlPreviewFile} className="object-contain" />
+                )}
+              </div>
+            )}
+          </div>
+
+          <div className="mb-5 flex flex-col w-full">
+            <input
+              required
+              type="file"
+              id="inputFile"
+              name="inputFile"
+              className="hidden"
+              accept=".png, .jpg, .jpeg, .svg, .gif, .mp3, .wav, .ogg, .mp4, ."
+              onChange={(e) => fileHandleChange(e)}
+            />
+          </div>
+        </div>
+
+        <div>
+
+          {/* ===> TITLE */}
+          <div className="">
+            <input required type="text" name="title" onChange={e => setTitle(e.target.value)} className="form-control" />
+            <label htmlFor="title">Title</label>
+          </div>
+
+          {/* ===> DESCRIPTION */}
+          <div className="">
+            <textarea required maxLength="2490" name="description" onChange={e => setDescription(e.target.value)} className="form-control"></textarea>
```

Comment:
Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/esolidar/web3/pullRequest?prid=9542591)
```suggestion
            <textarea required maxLength="2490" name="description" onChange={e => setDescription(e.target.value)} className="form-control" />
```